pkg/cloudprovider/azure.go

```diff
@@ -34,13 +34,14 @@ const (
 // to the Azure cloud API
 type Azure struct {
 	CloudProvider
-	resourceGroup        string
-	env                  azure.Environment
-	vmClient             compute.VirtualMachinesClient
-	virtualNetworkClient network.VirtualNetworksClient
-	networkClient        network.InterfacesClient
-	nodeMapLock          sync.Mutex
-	nodeLockMap          map[string]*sync.Mutex
+	resourceGroup            string
+	env                      azure.Environment
+	vmClient                 compute.VirtualMachinesClient
+	virtualNetworkClient     network.VirtualNetworksClient
+	networkClient            network.InterfacesClient
+	applicationSecurityGroup network.ApplicationSecurityGroup
```

Comment:
Is the new `Azure.applicationSecurityGroup` used anywhere?

Reply:
no. removed in latest update.